### PR TITLE
fix Analytics unit tests for mulitple events

### PR DESCRIPTION
**Pull request by huminzhi**

### CODE REVIEW

File: libs/SalesforceAnalytics/SalesforceAnalyticsTests/EventStoreManagerTests.m [hunk 1]
```diff
@@ -105,23 +105,6 @@ - (void) testFetchOneEvent {
 }
 
 /**
- * Test for fetching all stored events.
- */
-- (void) testFetchAllEvents {
```
Comment (huminzhi) [inline libs/SalesforceAnalytics/SalesforceAnalyticsTests/EventStoreManagerTests.m]: This test is same as testStoreMultipleEvents, so I removed this one, and rename testStoreMultipleEvents to testStoreAndFetchMultipleEvents

File: libs/SalesforceAnalytics/SalesforceAnalyticsTests/EventStoreManagerTests.m [hunk 2]
```diff
@@ -87,8 +87,8 @@ - (void) testStoreMultipleEvents {
     NSArray<SFSDKInstrumentationEvent *> *events = [self.storeManager fetchAllEvents];
     XCTAssertTrue(events != nil, @"List of events should not be nil");
     XCTAssertEqual(2, events.count, @"Number of events stored should be 2");
-    XCTAssertTrue([event1 isEqual:[events firstObject]], @"Stored event should be the same as generated event");
```
Comment (huminzhi) [inline libs/SalesforceAnalytics/SalesforceAnalyticsTests/EventStoreManagerTests.m]: when fetch events from file system, it's not ordered by time stamp, so cannot guarantee the events are as the same sequence as created, thus use [events containsObject: ] to make sure the event exists in the store

File: libs/SalesforceAnalytics/SalesforceAnalyticsTests/EventStoreManagerTests.m [hunk 3]
```diff
@@ -156,8 +139,6 @@ - (void) testDeleteMultipleEvents {
     NSArray<SFSDKInstrumentationEvent *> *eventsBeforeDel = [self.storeManager fetchAllEvents];
     XCTAssertTrue(eventsBeforeDel != nil, @"List of events should not be nil");
     XCTAssertEqual(2, eventsBeforeDel.count, @"Number of events stored should be 2");
-    XCTAssertTrue([event1 isEqual:[eventsBeforeDel firstObject]], @"Stored event should be the same as generated event");
```
Comment (huminzhi) [inline libs/SalesforceAnalytics/SalesforceAnalyticsTests/EventStoreManagerTests.m]: this flow was verified in previous test, no need to check here again.

**Comment by bhariharan** [comment]
👍

**Pull request merged**